packages/ggit/src/flat-ggit-error.ts

```diff
@@ -0,0 +1,82 @@
+import type { ModelRunErrorCollection, ErrorResponse } from './output';
+
+export enum GGITErrorClassification {
+  INPUT_VALIDATION_ERROR = 'InputValidationError',
+  API_BUILD_ERROR = 'APIBuildError',
+  ERROR = 'Error',
+}
```

Comment:
These are the three categories of error that we receive from soil metrics. Since this is soil-metrics specific, I felt it made more sense to declare here vs in @nori-dot-com/errors